### PR TITLE
fix(TextArea): better accessibility

### PR DESCRIPTION
**Pull request by savutsang**
DS-767
DS-754

Just accessibility changes

### CODE REVIEW

File: packages/react/src/components/text-area/text-area.tsx [hunk 1]
```diff
@@ -159,11 +160,17 @@ export const TextArea: VoidFunctionComponent<TextAreaProps> = ({
                 <Counter
                     data-testid="char-counter"
                     aria-live="polite"
-                    aria-hidden="true"
                     id={idCounter}
                     valid={inputValueLength <= maxLength}
                 >
-                    {t('characters', { current: inputValueLength, max: maxLength })}
+                    {inputValueLength}
```
Comment (savutsang) [inline packages/react/src/components/text-area/text-area.tsx:166]: @LarryMatte I was thinking if this way would be the same and better?

```
<span aria-live="polite" aria-label="12 of 30 characters available">
  12 / 30 characters
</span>
```

Comment (savutsang) [reply packages/react/src/components/text-area/text-area.tsx:166]: BTW I just had this though immediately after doing my PR

Comment (pylafleur) [reply packages/react/src/components/text-area/text-area.tsx:166]: That would improve code readability quite a bit!

**Review by pylafleur — COMMENTED** [review]
I guess we'll wait for Larry for comments on the suggestion. As for the uuid, I think something should probably be done. What do others think?

File: packages/react/src/components/text-area/text-area.test.tsx.snap [hunk 2]
```diff
@@ -121,13 +121,16 @@ input + .c1 {
...
+    for="UUID"
   >
     Comment (required)
   </label>
   <textarea
+    aria-describedby="UUID_hint UUID_invalid UUID"
     class="c3"
     data-testid="textarea"
     disabled=""
+    id="UUID"
```
[the remark this answers is not in the document]
Comment (savutsang) [reply packages/react/src/components/text-area/text-area.test.tsx.snap:133]: The mock could generate a new value everytime uuid is called, but in a consistent way.

uuid1
uuid2
uuid3

[the remark this answers is not in the document]
Comment (pylafleur) [reply packages/react/src/components/text-area/text-area.test.tsx.snap:133]: Yep, that would work! 👍

**Pull request closed without merging**